stacker_blueprints/sqs.py

```diff
@@ -25,6 +26,14 @@ def validate_queue(queue):
     if "RedrivePolicy" in queue:
         queue["RedrivePolicy"] = sqs.RedrivePolicy(**queue["RedrivePolicy"])
 
+    if "FifoQueue" in queue and queue["FifoQueue"]:
```

Comment:
I think what you want here is:

```
if queue.get("FifoQueue"):
```

`.get` on a dict returns None if the key doesn't exist.